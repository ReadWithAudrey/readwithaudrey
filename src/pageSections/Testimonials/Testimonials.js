import React from 'react'
import StyledTestimonials from './Testimonials.style'
import { Button, TextBox } from '../../components/index'
import { Link } from 'gatsby'
import PropTypes from 'prop-types'

const Testimonials = () => (
  <StyledTestimonials>
    <h1 className="f2 pink tc">What our community are saying</h1>
    <TextBox>
      &quot;The reading sessions are getting longer each week because we find
      that there&apos;s so much to talk about!&quot; Cristina, Sao Paulo.
    </TextBox>

    <TextBox>
      &quot;I like the fact that I can talk about the book with a person that I
      don&apos;t know - I can be honest and don&apos;t feel like I need to
      impress anyone.&quot; Shreya, Bangalore.
    </TextBox>

    <TextBox>
      &quot;You’re not only discovering new books, you’re having a chance to
      look at the book from a completely different perspective&quot;. Matt,
      London.
    </TextBox>
    <TextBox>
      &quot;I loved my experience with Sara and I think she would agree that
      we’ve bonded in a very strong way. We’ve kept in touch&quot;. Maria,
      Mexico City.
    </TextBox>

    <TextBox>
      &quot;Things are going great and I am enjoying the reading sessions. I
      think we will be ready this week for book #3 if you can believe it!&quot;
      Stefanie, Chicago.
    </TextBox>

    <p className="f4 tc">
      Every week people are reading to each other all around the world.
    </p>
    <p className="f4 tc">Are you ready to join the Audrey community?</p>
    <Link to="/Form1/" className="no-underline white w-100">
      <Button type="register">Get started</Button>
    </Link>
  </StyledTestimonials>
)

Testimonials.propTypes = {
  siteTitle: PropTypes.string,
}

export default Testimonials
