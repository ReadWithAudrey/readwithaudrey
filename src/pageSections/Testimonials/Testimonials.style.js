import styled from 'styled-components'

const StyledTestimonials = styled.div.attrs({
  className:
    'bg-light-green flex flex-column items-center justify-between ph4 ph6-ns pv4 page-padding montserrat w-100',
})``
export default StyledTestimonials
