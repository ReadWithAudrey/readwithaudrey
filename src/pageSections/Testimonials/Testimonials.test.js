import React from 'react'
import renderer from 'react-test-renderer'
import Testimonials from './Testimonials'

describe('Examples component snapshot test', () =>
  it('renders correctly', () => {
    const tree = renderer.create(<Testimonials />).toJSON()
    expect(tree).toMatchSnapshot()
  }))
