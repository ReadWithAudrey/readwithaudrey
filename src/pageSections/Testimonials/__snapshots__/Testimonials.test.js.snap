// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Examples component snapshot test renders correctly 1`] = `
<div
  className="sc-bdVaJa bg-light-green flex flex-column items-center justify-between ph4 page-padding montserrat w-100 bDWFJH"
>
  <h1
    className="f2 pink tc"
  >
    What our community are saying
  </h1>
  <p
    className="sc-chPdSV montserrat f5 tc lh-copy AOtmu"
  >
    "The reading sessions are getting longer each week because we find that there's so much to talk about!" Cristina, Sao Paulo.
  </p>
  <p
    className="sc-chPdSV montserrat f5 tc lh-copy AOtmu"
  >
    "I like the fact that I can talk about the book with a person that I don't know - I can be honest and don't feel like I need to impress anyone." Shreya, Bangalore.
  </p>
  <p
    className="sc-chPdSV montserrat f5 tc lh-copy AOtmu"
  >
    "You’re not only discovering new books, you’re having a chance to look at the book from a completely different perspective". Matt, London.
  </p>
  <p
    className="sc-chPdSV montserrat f5 tc lh-copy AOtmu"
  >
    "I loved my experience with Sara and I think she would agree that we’ve bonded in a very strong way. We’ve kept in touch". Maria, Mexico City.
  </p>
  <p
    className="sc-chPdSV montserrat f5 tc lh-copy AOtmu"
  >
    "Things are going great and I am enjoying the reading sessions. I think we will be ready this week for book #3 if you can believe it!" Stefanie, Chicago.
  </p>
  <p
    className="f4 tc"
  >
    Every week people are reading to each other all around the world. Are you read to join them?
  </p>
  <Link
    className="no-underline white w-100"
    to="/Form1/"
  >
    <button
      className="sc-gqjmRU bg-pink hover-bg-orange white pa2 br2 w-40 w-30-ns h3 f3 mt4 center kKOnBs"
    >
      Get started
    </button>
  </Link>
</div>
`;
